test/algo/compression/pruning/test_speedup_v2_init_mask.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import unittest
+
+import functools
+import torch
+import torch.nn.functional as F
+
+from nni.common.concrete_trace_utils import concrete_trace
+from nni.compression.pytorch.pruning import L1NormPruner
+from nni.compression.pytorch.speedup.compress_modules import no_replace
+from nni.compression.pytorch.speedup.v2 import ModelSpeedup
+from nni.compression.pytorch.utils import count_flops_params
+
+def simple_annotation():
+    def simple_wrapper(old_func):
+        @functools.wraps(old_func)
+        def new_func(*args, **kwargs):
+            output = old_func(*args, **kwargs)
+            return output
+        return new_func
+    return simple_wrapper
+
+class WithAnno1(torch.nn.Module):
+    """
+    test for annotations
+    """
+    def __init__(self):
+        super().__init__()
+        self.relu = torch.nn.ReLU6()
+
+    @simple_annotation()
+    def forward(self, input: torch.Tensor) -> torch.Tensor:
+        return self.relu(input)
+
+class WithAnno2(torch.nn.Module):
+    """
+    test for annotations
+    """
+    def __init__(self):
+        super().__init__()
+        self.relu = torch.nn.ReLU6()
+
+    @simple_annotation()
+    @simple_annotation()
+    def forward(self, input: torch.Tensor) -> torch.Tensor:
+        return self.relu(input)
+
+class WithAnno3(torch.nn.Module):
+    """
+    test for annotations
+    """
+    def __init__(self):
+        super().__init__()
+        self.relu = torch.nn.ReLU6()
+
+    @simple_annotation()
+    @simple_annotation()
+    @simple_annotation()
+    def forward(self, input: torch.Tensor) -> torch.Tensor:
+        return self.relu(input)
+
+class TorchModel(torch.nn.Module):
+    def __init__(self, relu):
+        super().__init__()
+        self.conv1 = torch.nn.Conv2d(1, 20, 5, 1)
+        self.relu1 = relu
+        self.conv2 = torch.nn.Conv2d(20, 50, 5, 1)
+        self.fc1 = torch.nn.Linear(4 * 4 * 50, 500)
+        self.fc2 = torch.nn.Linear(500, 10)
+        self.relu2 = torch.nn.ReLU6()
+        self.relu3 = torch.nn.ReLU6()
+        self.max_pool1 = torch.nn.MaxPool2d(2, 2)
+        self.max_pool2 = torch.nn.MaxPool2d(2, 2)
+
+    def forward(self, x):
+        x = self.relu1(self.conv1(x))
+        x = self.max_pool1(x)
+        x = self.relu2(self.conv2(x))
+        x = self.max_pool2(x)
+        x = x.view(-1, x.size()[1:].numel())
+        x = self.relu3(self.fc1(x))
+        x = self.fc2(x)
+        return F.log_softmax(x, dim=1)
+
+# class InitMaskTestCase(unittest.TestCase):
```

Comment:
why is this part commented out?

Reply:
pre-set in/out masks for speedup has some issues in previous implementation, so I comment the relation code and this test, will add this test back after the pre-set in/out masks feature done.